arrays/js/arrays.js

```diff
@@ -0,0 +1,95 @@
+function isArray1(a) {
+  return (a instanceof Array);
+}
+
+function isArray2(a) {
+  return Array.prototype.isPrototypeOf(a);
+}
+
+function range(a, b, c) {
```

Comment:
next cases should work:
range(10) [0..9]
range(1, 10) [1..9]
range(1, 10, 3) [1, 4, 7]
range(10, null, 3) [0, 3, 6, 9]
range(10, null) the same with range(10)
range(-2, -5) []
range(-5, -2) [-5, -4, -3]
range(-5, -2, 2) [-5, -3]
range(-5, -2, -2) []
range(-5, null, 2) []
range(-5, null, -2) [0, -2, -4]
range(-10, -20, -5) [-10, -15]
range(-20, -10, -5) []

FYI this function can be implemented ~10 lines, firstly you should find step and length of result array (the difficult part of this function)